### PR TITLE
Created VYE DGI Service - Reverts #19692

### PR DESCRIPTION
**Pull request by RachalCassity**
Reverts department-of-veterans-affairs/vets-api#19692

### CODE REVIEW

**Comment by rjohnson2011** [comment]
Just verifying all the manifest certs are in per Trevor's comment -> https://github.com/department-of-veterans-affairs/vets-api/pull/19592#issuecomment-2515602646

**Comment by bosawt** [comment]
> Just verifying all the manifest certs are in per Trevor's comment -> [#19592 (comment)](https://github.com/department-of-veterans-affairs/vets-api/pull/19592#issuecomment-2515602646)

Actually updating my comment, it looks like the cert was defined, but JWT encoding expects a private key. The format used in the fixture here is the correct one, so manifest values need to match this type of file

**Comment by RachalCassity** [comment]
@bosawt Yes, thank you. I saw that as well. I went ahead and corrected the paths for the team in this [PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3301).

**Review by rjohnson2011 — DISMISSED** [review]

**Comment by RachalCassity** [comment]
Still waiting for the team to get the following parameters added to AWS:
- sandbox dgib ica11-rca2
- sandbox dgib jwt
- sandbox dgib jwt
- prod dgib ica11-rca2

**Comment by GcioGregg** [comment]
I added a fake combined VA cert in modules/vye/spec/fixtures and pointed to that in settings.yml, replacing the empty string.

File: lib/vye/dgib/authentication_token_service.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Vye
+  module DGIB
+    class AuthenticationTokenService
+      ALGORITHM_TYPE = 'RS256'
```
Comment (ericboehs) [inline lib/vye/dgib/authentication_token_service.rb:6]: Should this be RSA256?

Comment (GcioGregg) [reply lib/vye/dgib/authentication_token_service.rb:6]: My understanding is that RS256 is the valid algorithm_type for jwt (RSA using SHA256)

**Review by GcioGregg — APPROVED** [review]

**Review by ericboehs — APPROVED** [review]

**Pull request merged**